### DISCUSSION TITLE
Text color for modules

### DISCUSSION THREAD

**Discussion by mihaifm** [catppuccin/tmux#186] [Help] [answered]
Hi, thanks for this great plugin. Can you tell me which variable controls the text color for modules? The docs don't make it very clear.

**Comment by vdbe** [comment, marked as answer]
The text color of a module is `$thm_fg` wich is set in `./catppuccin-<theme>.tmuxtheme` chosen by `@catppuccin_flavour`.

**Reply by vdbe** [reply]
You can overwrite it by using the default tmux syntax in https://github.com/catppuccin/tmux?tab=readme-ov-file#override-the-specific-module-text

**Reply by mihaifm** [reply]
Great, thanks for the reply.
